lib/sentry/finch_client.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Sentry.FinchClient do
+  @behaviour Sentry.HTTPClient
+
+  @moduledoc """
+  The built-in HTTP client.
```

Comment:
Shows up correctly in the module doc summary with ExDoc:

```suggestion
  The built-in HTTP client.

```